component/authenticator/src/main/java/org/wso2/carbon/identity/application/authenticator/totp/internal/TOTPDataHolder.java

```diff
@@ -76,4 +78,14 @@ public void setConfigurationContextService(
 			ConfigurationContextService configurationContextService) {
 		this.configurationContextService = configurationContextService;
 	}
-}
\ No newline at end of file
+
+	public IdentityEventService getIdentityEventService() {
```

Comment:
Better to have a new line after the method signature to comply with other methods.

Reply:
Fix : https://github.com/wso2-extensions/identity-outbound-auth-totp/pull/63